app/controllers/flowchart_controller.rb

```diff
@@ -1,4 +1,93 @@
 # frozen_string_literal: true
 
 class FlowchartController < ApplicationController
+  # TODO: Add Error Checking for invalid inputs or missing data in the database (eg. no flowchart with that id)
+
+  def create
+    begin
+        @flowchart = Flowchart.create(JSON.parse(request.body.read))
+    rescue
+        render status: 400, json: { error: "Could not create flowchart." }
```

Comment:
I do this because I think it's better than having rails send a 500 error filled with column names and potentially sensitive data to the client in the response.

Reply:
I don't think 400 is the appropriate error code either because that usually implies that the User supplied invalid data, this should be something we use when they like enter an empty string or empty payload or something. 

I think we should still return a 500 if it's a valid payload, so that we all know it's our problem and not theirs.

Reply:
Done